src/index.js

```diff
@@ -0,0 +1,115 @@
+let defaultLogger = console;
+
+/**
+  @class Catchie
+  Consume errors and retry the function execution given times
+
+  Supported environment variables:
+  MAX_REPEAT - cap amount of function retry
+**/
+class Catchie {
+  /**
+    @constructor
+    Construct instance of the module
+    @param logger {Object} - logger instnace
+  **/
+  constructor(logger) {
+    this.logger = [undefined, null].includes(logger) ? defaultLogger : logger;
+    this.successCount = 0;
+    this.failCount = 0;
+  }
+
+  /**
+    @function clear
+    Reset the values of successCount and failureCount
+  **/
+  clear() {
+    this.successCount = 0;
+    this.failureCount = 0;
+  }
+
+  /**
+    @function callCount
+    Get the total amount of given function calls
+    @return {number} - total amount of function calls
+  **/
+  get callCount() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide a setter matching this getter for 'callCount'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3AS2376)